eZ/Publish/Core/FieldType/FieldType.php

```diff
@@ -271,7 +271,7 @@ public function applyDefaultSettings(&$fieldSettings)
      */
     protected function getSortInfo(Value $value)
     {
-        throw new \RuntimeException('Not implemented, yet.');
+        return null;
```

Comment:
This change is small but IMHO we should extract it to separate PR because is not only related to Generic Field Type